sdk/search/search-documents/review/search-documents.api.md

```diff
@@ -221,15 +223,25 @@ export interface CustomAnalyzer {
     name: string;
     odatatype: "#Microsoft.Azure.Search.CustomAnalyzer";
     tokenFilters?: string[];
-    tokenizerName: string;
+    tokenizer: string;
```

Comment:
was this an intentional rename from `tokenizerName`? My understanding of this PR is that we're switching up the codegen, not renaming public properties?

Reply:
This is the only exception. I could have overridden so this name change does not happen. But, since the next release is GA and we are introducing breaking changes anyway and the new name reflects swagger more closely and looks more clean, I did not override it.